gradle/jacoco.gradle

```diff
@@ -56,15 +56,50 @@ project.afterEvaluate {
                         "src/$productFlavorName/java",
                         "src/$buildTypeName/java",
                         "src/commonMain/kotlin",
-                        "src/androidLibMain/lotlin"
```

Comment:
🤔

Reply:
ah I fixed that and then reverted it :D I should see what else I reverted, thanks.

Reply:
oh ok it's already fixed 🥇